models/stats.go

```diff
@@ -54,3 +63,46 @@ func (s *Stats) Scan(value interface{}) error {
 
 	return json.Unmarshal(b, &s)
 }
+
+func GetRegions() []string {
+	if RegionsLastUpdate.IsZero() && time.Since(RegionsLastUpdate).Seconds() < 60 {
+		// return cached list of regions
+		return Regions
+	}
+
+	catalystJSONURL, exists := os.LookupEnv("CATALYSTS_JSON")
+	if !exists {
+		catalystJSONURL = "https://livepeer.github.io/livepeer-infra/catalysts.json"
+	}
+
+	resp, err := http.Get(catalystJSONURL)
+	if err != nil {
+		log.Printf("Can't fetch the %s: %s", catalystJSONURL, err)
+		return Regions // return previosuly cached list of regions
+	}
+	defer resp.Body.Close()
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		log.Printf("Can't read the %s: %s", catalystJSONURL, err)
+		return Regions // return previosuly cached list of regions
+	}
+
+	type catalystEnvData struct {
+		Counts    map[string]int `json:"counts"`
+		Urls      []string       `json:"urls"`
+		GlobalUrl string         `json:"global_url"`
+	}
+	var catalystData map[string]catalystEnvData
+	err = json.Unmarshal(body, &catalystData)
+	if err != nil {
+		log.Printf("Can't parse the %s: %s", catalystJSONURL, err)
+		return Regions // return previosuly cached list of regions
+	}
+	Regions = []string{}
+	for region := range catalystData["prod"].Counts {
```

Comment:
IMO it's ok, because we don't have staging for orch-tester jobs. And I think it does make sense to introduce it, because:
1) We don't have many issues with leaderboard service (and orch-tester)
2) It's not critical for our infra
3) Setting up staging orch-tester would mean that we need to run it somewhere and each orch-tester job costs us money; one idea would be that orch-tester runs once and then send data to both prod and staging (but I don't see too much need for that)